docs/AnaplanExport-batchsource.md

```diff
@@ -53,16 +51,7 @@ It can be found on the Dashboard in the Google Cloud Platform Console.
 **File archive name in GCS bucket:** The desired file name for the data export from Anaplan model.
 
 **Format:** Format of the data that export from the Anaplan model.
-The format must be one of 'avro', 'blob', 'csv', 'delimited', 'json', 'parquet', 'text', 'tsv', or the
-name of any format plugin that you have deployed to your environment.
-If the format is a macro, only the pre-packaged formats can be used.
-If the format is 'blob', every input file will be read into a separate record.
-The 'blob' format also requires a schema that contains a field named 'body' of type 'bytes'.
-If the format is 'text', the schema must contain a field named 'body' of type 'string'.
-
-**Delimiter:** Delimiter to use when the format is 'delimited'. This will be ignored for other formats.
-
-**Skip Header** Whether to skip the first line of each file. Supported formats are 'text', 'csv', 'tsv', 'delimited'.
+The format must be one of 'csv' and 'tsv'.
```

Comment:
how come format was kept? Anaplan doesn't export in a standard format?

Reply:
The way how Anaplan export is to generate a csv or tsv file. And the file is downloaded into GCS as a binary file. So plugin would have to read the file as it is.

Reply:
there isn't a way to tell Anaplan which format to use? Would be great if the user didn't have to know the format.

Reply:
Ha, no. Anaplan is able to generate file with a column mix with integer and decimal.
If we want to pick a best match, it would be decimal.

Reply:
question here wasn't about the numeric types, but the format of the file that gets written to GCS. There isn't a way to tell Anaplan to always use tsv for example?